webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -330,7 +347,9 @@ const NavigationSideBar: React.FC = () => {
             <NavigationItem
               icon={settings.navbarCollapsed ? 'expand' : 'collapse'}
               label={settings.navbarCollapsed ? 'Expand' : 'Collapse'}
-              tooltip={settings.navbarCollapsed}
+              tooltip={
+                settings.navbarCollapsed ? 'Expand (Ctrl/Cmd + U)' : 'Collapse (Ctrl/Cmd + U)'
```

Comment:
**sugg**: Suggest making this adapt to remapped shortcut as well

Reply:
Also just fyi you have it mapped to `Ctrl + Shift + u` which is the normal way to write that.

Reply:
I'll also add: it should only display the relevant key for the operating system.

Reply:
To be clear, for Windows, we want `Ctrl + U` to be the default shortcut, while for Mac, we want `Cmd + U` to be the default shortcut? In another word, we want pressing `Ctrl` on Windows to be the same effect as pressing `Cmd` for Mac?